lib/developer_portal/lib/liquid/template/resolver.rb

```diff
@@ -26,7 +26,7 @@ def initialize(scope)
       end
 
       def _find_all(name, prefix, partial, details, key = nil, locals = [])
-        path = Path.build(name, prefix, partial)
+        path = ActionView::TemplatePath.build(name, prefix, partial)
```

Comment:
Fixes the error: 
```
uninitialized constant Liquid::Template::Resolver::Path
Did you mean?  XPath
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/activesupport-7.1.5.1/lib/active_support/deprecation/constant_accessor.rb:39:in `const_missing'
/opt/ci/workdir/lib/developer_portal/lib/liquid/template/resolver.rb:29:in `_find_all'
```

Reply:
Do you remember whether the class/module was removed with the upgrade or something else?

Reply:
If you don't remember I think it is fine.

Reply:
Some references:
https://github.com/rails/rails/pull/42195/files#diff-82dd8abd58264f9fe9de27f3be5016cca3e4722346825f39419e6c596e54ad9fR13-R14

https://guides.rubyonrails.org/7_1_release_notes.html#action-view-removals

So, yes, `ActionView::TemplatePath` is an equivalent.